src/foolscap/banana.py

```diff
@@ -10,31 +11,47 @@
 from foolscap.slicers.allslicers import RootSlicer, RootUnslicer
 from foolscap.slicers.allslicers import ReplaceVocabSlicer, AddVocabSlicer
 
-import stringchain
-import tokens
-from tokens import SIZE_LIMIT, STRING, LIST, INT, NEG, \
-     LONGINT, LONGNEG, VOCAB, FLOAT, OPEN, CLOSE, ABORT, ERROR, \
-     PING, PONG, \
-     BananaError, BananaFailure, Violation
+from . import stringchain
+from . import tokens
+from .tokens import (SIZE_LIMIT, STRING, LIST, INT, NEG, 
+                    LONGINT, LONGNEG, VOCAB, FLOAT, OPEN, CLOSE, ABORT, ERROR, 
+                    PING, PONG, 
+                    BananaError, BananaFailure, Violation)
 
 EPSILON = 0.1
 
+if six.PY3:
+    long = int
+
+def ensure_byte(ch):
+    # PY3PORT - new function
+    if six.PY2:
+        return ch
+    elif six.PY3:
+        return six.int2byte(ch)
+
 def int2b128(integer, stream):
     if integer == 0:
```

Comment:
int2b128 did not have docstring already. Since its a port, I wont be adding docstrings for entities which did not have it already for keeping the changes efficient. For any new functions I add, yes, I will be adding docstrings.

Reply:
I think functions that get changed can benefit from a docstring, too. Then reviewers don't have to (also) convince themselves what the function is supposed to be doing..Obviously, we have to limit scope somewhere (can't fix everything at once), but...